pkg/process/config/yaml_config.go

```diff
@@ -159,13 +154,20 @@ func (a *AgentConfig) loadSysProbeYamlConfig(path string) error {
 	}
 
 	if config.Datadog.GetBool(key(spNS, "enable_tcp_queue_length")) {
+		a.EnableSystemProbe = true
```

Comment:
Should we emit a log message as with security and network?

Reply:
Replied below, this is done!